mappings/aaf267c77f09a5b4f006009a3235f3a9765660ccf2e6e1e0380303d54b65726d697420746865204675636b.json

```diff
@@ -0,0 +1,64 @@
+{
+    "subject": "aaf267c77f09a5b4f006009a3235f3a9765660ccf2e6e1e0380303d54b65726d697420746865204675636b",
+    "url": {
+        "sequenceNumber": 0,
+        "value": "https://kermitthefuck.com",
```

Comment:
Hi @mrsqurriel,

The website is still not working. Please refer to #4249 

Thanks!

Reply:
Forgot to sign, finalize and update before. I will submit a new PR